Sprint-3/reading-list/script.js

```diff
@@ -21,3 +21,26 @@ const books = [
   },
 ];
 
+let list = document.querySelector("#reading-list");
+
+books.forEach(item => {
+  let book = document.createElement('li')
```

Comment:
you might want to add some css here to remove the bullet point

Reply:
Thank you, done )